routes/web.php

```diff
@@ -84,6 +84,7 @@
         $router->post('/store-camp-newsfeed','NewsFeedController@storeNewsFeed');
         $router->post('/update-camp-newsfeed','NewsFeedController@updateNewsFeed');
         $router->post('thread/save','ThreadsController@store');
+        $router->put('thread/update/{id}','ThreadsController@update');
```

Comment:
This is the error that we made in 2.0. Instead of id it should be title of the thread. So that it's more SEO friendly